artifactory/commands/utils/questionnaire.go

```diff
@@ -276,3 +278,13 @@ func OptionalKeyCallback(iq *InteractiveQuestionnaire, key string) (value string
 	}
 	return value, err
 }
+
+func ValidateMapEntry(key string, value interface{}, writersMap map[string]AnswerWriter) error {
```

Comment:
This function does not belong to the questionnaire module.
We need to find a better place to put it ...

Reply:
Well, you know more than me about this, where should we put it?